requirements/edx-sandbox/base.txt

```diff
@@ -4,8 +4,8 @@
 #
 #    make upgrade
 #
-common/lib/sandbox-packages
-common/lib/symmath
+-e file:///edx/app/edxapp/edx-platform/common/lib/sandbox-packages
+-e file:///edx/app/edxapp/edx-platform/common/lib/symmath
```

Comment:
@feanil this change is made by `make upgrade`, does this look fine?

Reply:
It looks like `make upgrade` did not run fully successfully.  This change is usually fixed by the end of make upgrade.  So take a look at the output and see if anything went rong.